packages/hash/shared/src/prosemirror.ts

```diff
@@ -329,7 +327,9 @@ export const mutateSchema = (
     set nodes(newNodes) {
       for (const [key, value] of Object.entries(newNodes)) {
         if (!this.nodes[key]) {
+          // @ts-expect-error -- NodeType#schema is readonly in prosemirror-model
           value.schema = schema;
+          // @ts-expect-error -- NodeType#nodes is readonly in prosemirror-model
           this.nodes[key] = value;
```

Comment:
@nathggns do you see any possible issues with these `@ts-expect-error` instances? There are two more just below.